apps/parsley/.gitignore

```diff
@@ -6,6 +6,7 @@ cypress/screenshots/
 /src/**/*.css
 build.txt
 build.txt-e
+vite.config.ts.timestamp-*
```

Comment:
What is this?

Reply:
it's from a vite bug: https://github.com/vitejs/vite/issues/13267